test/integration/etcd_storage_path_test.go

```diff
@@ -992,7 +992,7 @@ func (c *allClient) verb(verb string, gvk unversioned.GroupVersionKind) (*restcl
 	if err != nil {
 		return nil, err
 	}
-	return restclient.NewRequest(c.client, verb, baseURL, versionedAPIPath, contentConfig, *serializers, c.backoff, c.config.RateLimiter), nil
+	return restclient.NewRequest(c.client, verb, baseURL, versionedAPIPath, contentConfig, *serializers, c.backoff, c.config.RateLimiter, 0), nil
```

Comment:
Sorry that I'm just now getting to the point of chasing this down, but why not just call `restclient.NewRequest().Timeout(0)`?

Reply:
Here, it doesn't matter. What I care is that the `--request-timeout` flag is being properly propagated to the request.